napari_btrack/constants.py

```diff
@@ -0,0 +1,30 @@
+"""
+This modules constains variables that are used throughout the
+napari_btrack package.
+"""
```

Comment:
I like this

Reply:
I just spotted some typos in that docstring, so I've fixed them